GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -48,6 +48,10 @@ public protocol SceneProvider: AnyObject {
     func windowScene() -> UIWindowScene?
 }
 
+private extension Glia {
+    static let maximumUploads = 25
```

Comment:
Where does this value come from? Is it a backend limitation or it's client imposed?

Reply:
Honestly I don't know where it initially came from, we have this limitation for a long time

Reply:
yes, it is backend limitation, however, this value doesn't come form backend.